src/coreclr/jit/morph.cpp

```diff
@@ -10709,7 +10709,7 @@ GenTree* Compiler::fgOptimizeHWIntrinsic(GenTreeHWIntrinsic* node)
             GenTree* op1 = node->Op(1);
             GenTree* op2 = node->Op(2);
 
-            if (!op1->OperIsHWIntrinsic(NI_Sve_CreateTrueMaskAll) &&
+            if (!op1->OperIsHWIntrinsic(NI_Sve_CreateTrueMaskAll) ||
```

Comment:
I think this is causing an incorrect morph.

```
fgMorphTree BB01, STMT00005 (before)
               [000029] DACXG------                         *  STORE_LCL_VAR long   V02 loc1         
               [000026] --CXG------                         \--*  HWINTRINSIC long   ubyte GetActiveElementCount
               [000028] -----------                            +--*  HWINTRINSIC mask   ubyte ConvertVectorToMask
               [000027] -----------                            |  +--*  HWINTRINSIC mask   ubyte CreateTrueMaskAll
               [000019] -----------                            |  \--*  LCL_VAR   simd16 V04 tmp1          (last use)
              x[000025] --CXG------                            \--*  HWINTRINSIC mask   ubyte ConvertVectorToMask
              x[000024] -----------                               +--*  HWINTRINSIC mask   ubyte CreateTrueMaskAll
              x[000021] --CXG------                               \--*  HWINTRINSIC simd16 ubyte LoadVector
               [000023] -----------                                  +--*  HWINTRINSIC mask   ubyte ConvertVectorToMask
               [000022] -----------                                  |  +--*  HWINTRINSIC mask   ubyte CreateTrueMaskAll
              x[000014] -----------                                  |  \--*  LCL_VAR   simd16<System.Numerics.Vector`1> V01 loc0          (last use)
              x[000114] -----------                                  \--*  AND       long  
               [000108] -----------                                     +--*  SUB       long  
               [000105] -----------                                     |  +--*  ADD       long  
               [000103] -----------                                     |  |  +--*  LCL_VAR   long   V13 tmp10         (last use)
               [000104] -----------                                     |  |  \--*  LCL_VAR   long   V14 tmp11        
               [000107] -----------                                     |  \--*  CNS_INT   long   1
               [000113] -----------                                     \--*  NOT       long  
               [000112] -----------                                        \--*  SUB       long  
               [000109] -----------                                           +--*  LCL_VAR   long   V14 tmp11         (last use)
               [000111] -----------                                           \--*  CNS_INT   long   1
```

In the above all nodes with an `x` are deleted.
It fires when `op2` is `000021 LoadVector` and deletes it.